website/docs/docs/deploy/advanced-ci.md

```diff
@@ -5,12 +5,12 @@ sidebar_label: "Advanced CI"
 description: "Advanced CI enables developers to compare changes by demonstrating the changes the code produces."
 ---
 
-# Advanced CI <Lifecycle status="preview" />
+# Advanced CI <Lifecycle status="enterprise"/>
```

Comment:
@reubenmc : should this have an enterprise pill or just compare changes section below?

Reply:
Let's just call all of Adv CI enterprise. I think we need to do some naming work here between compare changes vs advanced CI. But this is good for now!